### DISCUSSION TITLE
Question: Can't find app store

### DISCUSSION THREAD

**Discussion by whyteserious** [nocodb/nocodb#4353] [Q&A] [answered]
### Please confirm if bug report does NOT exists already ?

- [ ] I confirm there is no existing issue for this

### Steps to reproduce ?

Can't find app store

### Desired Behavior

where is app store

### Project Details

Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mysql2**
ProjectOnRootDB: **false**
RootDB: **mysql2**
PackageVersion: **0.98.3**

**Comment by wingkwong** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/35857179/200999004-10b7af51-6269-4089-961d-79fac43418ea.png)

![image](https://user-images.githubusercontent.com/35857179/200999026-9b641886-6b4e-411e-91d9-467fb7a61992.png)

**Comment by dstala** [comment]
@whyteserious `AppStore`, for security reasons, is now available only for Super-admin.
If you are super-admin & still unable to access it: it could be a migration bug. One of the users in discord got over it using the following steps

```
For others with the same error:
1. log in to the root_db (e.g. inside the docker container)
   1.1 Attach to the container: docker exec -it container_name_of_root_db /bin/bash
   1.2 Open execute MySQL CLI: mysql -u noco -p root_db  and enter the password for the noco user
2. Check the roles with SELECT email,roles FROM nc_users_v2;
3. Update the user you want to be super admin with the following command: UPDATE nc_users_v2 SET roles = 'user,super' WHERE email = 'myemail@example.com';
4. Recreate the containers (docker-compose up -d)
5. Logoff and Login
```
